x/merkledb/view_iterator_test.go

```diff
@@ -229,7 +229,7 @@ func Test_TrieView_Iterator_Random(t *testing.T) {
 	for iter.Next() {
 		expectedKey := uniqueKeys[i]
 		expectedValue := uniqueKeyChanges[expectedKey]
-		require.Equal([]byte(expectedKey), iter.Key())
+		require.True(bytes.Equal([]byte(expectedKey), iter.Key()))
```

Comment:
nit: why did we change this?

Reply:
[]byte{} or nil are equivalent for bytes.equal but not require.equal